symja_android_library/matheclipse-io/src/main/java/org/matheclipse/script/engine/MathScriptEngineFactory.java

```diff
@@ -46,30 +42,17 @@ public String getLanguageVersion() {
 
   @Override
   public String getMethodCallSyntax(String obj, String m, String... args) {
-    StringBuilder callSyntax = new StringBuilder(obj);
-    callSyntax.append(".").append(m).append("(");
-    for (int i = 0; i < args.length; i++) {
-      callSyntax.append(args[i]);
-      if (i < args.length - 2) {
```

Comment:
I think this was wrong and should have been `i <= args.length - 2` or `i < args.length - 1`.
As it was now the last two arguments were not separated but just concatenated.
Or do I miss something?